Easydict/Feature/ViewController/Window/WindowManager/EZWindowManager.m

```diff
@@ -904,11 +912,6 @@ - (void)closeFloatingWindow {
     self.floatingWindow.titleBar.pin = NO;
     [self.floatingWindow close];
     
-    if (![EZPreferencesWindowController.shared isShowing]) {
-        // recover last app.
-        [self activeLastFrontmostApplication];
```

Comment:
This is code is needed, we should active the last frontmost app when closing Easydict query window if the settings window is not currently displaying.

Does this code cause any problems?

Reply:
With this code, it will not work to use shortcut to show the settings window.

You could try to type 'cmd' + ',' when query window active, and check if settings window shown.

Reply:
This change seems not effect the user experience per my test.

Reply:
> With this code, it will not work to use shortcut to show the settings window.
> 
> You could try to type 'cmd' + ',' when query window active, and check if settings window shown.

I fixed this issue in 792769e3

Reply:
Activating the last frontmost app is mainly to restore the cursor position, making it convenient for users to continue using. 

Imagine a user is editing and modifying a word within a paragraph, such as forgetting how to spell "cursor", after invoking Easydict window to get the result, he wants to continue typing where he left off.

Reply:
Great! works well from my side.